src/views/Tenant/tenantTicketUtils.jsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react'
+import * as axios from 'axios'
+import Toast from '../../utils/toast'
+
```

Comment:
It doesn't look like a lot of this code is being referenced anywhere, did we have a plan to use these in places?

Reply:
The functions are used in ```src/views/Tenant/tenantTickets.jsx``` in the ```TenantTickets``` component. The functions in ```src/views/Tenant/tenantTicketUtils.jsx``` are used to create the functions needed for the ```TenantTickets``` component. Since most of these functions are essentially the same as those used in the Tickets page, I tried to find a way to reduce duplication, but couldn't directly reuse the function in the Tickets page.

Reply:
Ah I didn't see them sorry! 👍